File: program/convert-meters-to-miles/ConvertMetersToMiles.js
// program to convert meters to miles (1 mile = 1609.344 meters)

function convertMetersToMiles(meters) {
  return meters / 1609.344;
}

// const userInput=prompt('Input: ')
// alert(convertMetersToMiles(userInput))

const userInput = 8046.72;
console.log("input  : " + userInput);
console.log("output : " + convertMetersToMiles(userInput));

File: program/convert-meters-to-miles/convert-meters-to-miles.py
# --------- Program to convert meters to miles ---------#

# Covert meters to miles (1 mile = 1609 meters)
def metersToMiles(meter_value):
    return int(meter_value // 1609)


# User input
input_in_meters = float(input("Input  : "))
output_in_miles = metersToMiles(input_in_meters)
print("Output :", output_in_miles)
